### PR TITLE
Make warning markup consistent in CSS docs

### PR DESCRIPTION
**Pull request by wbamberg**
This is the companion to https://github.com/mdn/content/pull/7170: just as with notes, the Markdown converter is a bit picky about what it considers to be valid warnings. This PR is intended to make all the warnings in the CSS docs consistently marked up, so the converter will recognize them.

There were a couple of places where a note seemed more appropriate than a warning:

https://github.com/mdn/content/compare/main...wbamberg:fix-bad-warnings?expand=1#diff-0d0cb2f65ecb6218da10a536d655c903dcc79fafa43b842144c6a9c97f059cf9
https://github.com/mdn/content/compare/main...wbamberg:fix-bad-warnings?expand=1#diff-606aa6e4c8293078d1d882a01136522db8163df214fea744e81af659af2bd531

...and one where neither callout seemed appropriate:

https://github.com/mdn/content/compare/main...wbamberg:fix-bad-warnings?expand=1#diff-228b13687780e46984d06bcc8a4172fadfa92c00899354a97b782c82ebc1a9fe

### CODE REVIEW

File: files/en-us/web/css/@media/-moz-device-pixel-ratio/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 <p>The <strong><code>-moz-device-pixel-ratio</code></strong> Gecko-only <a href="/en-US/docs/Web/CSS">CSS</a> <a href="/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features">media feature</a> can be used to apply styles based on the number of device pixels per CSS pixel.</p>
 
 <div class="warning">
```
Comment (teoli2003) [inline files/en-us/web/css/@media/-moz-device-pixel-ratio/index.html:16]: Some are ` <div class="warning">`, some are `<div class="notecard warning">`.

Do we want/need to uniformize this too?

Comment (wbamberg) [reply files/en-us/web/css/@media/-moz-device-pixel-ratio/index.html:16]: I think we don't need to. If we were keeping this indefinitely, then yes `notecard warning` is correct. But just `warning` renders correctly, and for Markdown I think the only thing that matters is helping the converter recognize what it's looking at.

Comment (teoli2003) [reply files/en-us/web/css/@media/-moz-device-pixel-ratio/index.html:16]: Thanks, I was wondering if it is was needed for MD or not. Thanks. I will check the preview pages later.

Comment (wbamberg) [reply files/en-us/web/css/@media/-moz-device-pixel-ratio/index.html:16]: ...and I've checked locally that this PR makes the converter not issue warnings about warnings, and that the converted Markdown renders them properly too:

Screen Shot 2021-07-24 at 11 39 41 AM

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
\o/

**Pull request merged**